libs/detectors/x86/detector.py

```diff
@@ -1,3 +1,37 @@
+import json
+import os
+from pathlib import Path
+
+
+def get_or_create_model_json_file(config, device, camera_id):
```

Comment:
We will need to update these json files in the future with the latest version of the config file. So I think it would be good to split the creation part into another function, which does overwrite the json file if it already exists

Reply:
This function will be modified as we discuss in chat. :)